.eslintrc.js

```diff
@@ -1,6 +1,11 @@
 module.exports = {
     root: true,
     parser: '@babel/eslint-parser',
+    settings: {
+        react: {
+            version: 'detect',
+        },
+    },
```

Comment:
This change fixed React warning when I run the lint command. It simply auto-detects the installed react version

![image](https://user-images.githubusercontent.com/15683894/236735452-8995425d-21a3-43c8-aa72-3f356f9342c9.png)